src/System.Collections.Immutable/tests/ImmutableArrayBuilderTest.cs

```diff
@@ -288,10 +288,18 @@ public void Sort()
         [Fact]
         public void Sort_Comparison()
         {
-            var builder = new ImmutableArray<int>.Builder();
+            var builder = new ImmutableArray<int>.Builder(4);
```

Comment:
I could.  But the tests I added catch the case, regardless (the values get sorted with 0s at the beginning, causing the comparison to fail), so I'm not sure doing so would add much.